src/app/game/snakeladder/Dice.tsx

```
@@ -0,0 +1,49 @@
+    
+    // Simulate rolling animation
+    setTimeout(() => setRolling(false), 600);
+  };
+
+  return (
+    <div className={styles.diceContainer}>
+      <div 
+        className={`${styles.dice} ${rolling ? styles.rolling : ''}`} 
+        onClick={handleClick}
+        data-disabled={disabled || rolling}
+      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)